templates/CRM/Admin/Form/Setting/Miscellaneous.tpl

```diff
@@ -80,7 +80,8 @@
     <h3>{ts}reCAPTCHA Keys{/ts}</h3>
 
     <div class="description">
-      {ts 1="https://www.google.com/recaptcha"}reCAPTCHA is a free service that helps prevent automated abuse of your site. To use reCAPTCHA on public-facing CiviCRM forms: sign up at <a href="%1" "target=_blank">Google's reCaptcha site</a>; enter the provided public and private reCAPTCHA keys here; then enable reCAPTCHA under Advanced Settings in any Profile.{/ts}
+      {capture name="recaptcha_url" assign="recaptcha_url"}<a href="https://www.google.com/recaptcha" target="_blank">{ts}Google's reCaptcha site{/ts}</a>{/capture}
+      {ts 1=$recaptcha_url}reCAPTCHA is a free service that helps prevent automated abuse of your site. To use reCAPTCHA on public-facing CiviCRM forms: sign up at %1; enter the provided public and private reCAPTCHA keys here; then enable reCAPTCHA under Advanced Settings in any Profile.{/ts}
```

Comment:
Same as above, this was working OK, and while I'd favour removing "target=blank" from the actual string, I don't think it's worth breaking translations.